pubtools/exodus/_tasks/push.py

```diff
@@ -0,0 +1,127 @@
+import logging
+import subprocess
+import time
+
+from six.moves.urllib.parse import urljoin
+
+from pubtools.exodus.task import ExodusTask
+
+LOG = logging.getLogger("pubtools-exodus")
+LOG_FORMAT = "%(asctime)s [%(levelname)-8s] %(message)s"
+
+
+class ExodusPushTask(ExodusTask):
+    """Push a directory to the Exodus CDN"""
+
+    def __init__(self):
+        super(ExodusPushTask, self).__init__()
+        self.publish = None
+        self.src = None
+        self.dest = None
+        self.prefix = None
+
+    def add_args(self):
+        super(ExodusPushTask, self).add_args()
+
+        self.parser.add_argument(
+            "--source",
+            help="The path to the directory to be pushed to the exodus CDN",
+            nargs="+",
```

Comment:
Do you have a case in mind yet where multiple --source will be used?  I'm thinking:

- if it's not used in practice, this can be simplified by letting exodus-rsync manage the publish rather than adding our own code.
- if it is used on some large directories then maybe we should be letting exodus-rsync run concurrently rather than one process at a time?